content/en/blog/falcosidekick-response-engine-part-8-fluxv2.md

```diff
@@ -0,0 +1,346 @@
+* flux v0.16.0
+* gcloud v347.0.0
+
+## Tutorial
+
+### Provision local Kubernetes Cluster
+
+```shell
+$ minikube config view
+- cpus: 3
+- driver: virtualbox
+- memory: 8192
+
+minikube start
+```
+
+### Install Flux V2 - GitOps Toolkit
+
+I highly recommended that you check out [getting started](https://fluxcd.io/docs/get-started/) page of `Flux v2`. It gives you detailed installation instructions for Flux v2.
+
+For Flux v2 to create a GitHub repository for its resources, we have to define the token and username information. Then, Flux installed in a GitOps way, and Flux will push its manifest to the repository.
+
+```shell
+export GITHUB_USER=<username>
+export GITHUB_TOKEN=<token>
+```
+
+Once the necessary environment variables are defined, we can install Flux v2. The following command will create and push its manifests to the repository, then install Flux components.
+
+```shell
+flux bootstrap github \
+  --owner=$GITHUB_USER \
+  --repository=fleet-infra \
+  --branch=main \
+  --path=./clusters/my-cluster \
+  --personal
+```
+
+After the installation is complete, the next step is creating the alpine [GitRepository](./alpine-gitsource.yaml) and [Kustomization](./alpine-kustomization.yaml) CRD's (Custom Resource Definitions). For more information, see [components](https://fluxcd.io/docs/components/).
+
+Apply the CRD files as follows:
+
+```shell
+$ kubectl apply -f https://raw.githubusercontent.com/developer-guy/kubernetes-response-engine-based-on-flux-v2-gitops-toolkit/master/alpine-gitsource.yaml
+gitrepository.source.toolkit.fluxcd.io/alpine created
+
+$ kubectl apply -f https://raw.githubusercontent.com/developer-guy/kubernetes-response-engine-based-on-flux-v2-gitops-toolkit/master/alpine-kustomization.yaml
+kustomization.kustomize.toolkit.fluxcd.io/alpine created
+```
+
+Alternatively, we can use the [Flux CLI](https://fluxcd.io/docs/cmd/#installation).
+
+To create `GitRepository`:
+
+```shell
+flux create source git alpine \
+    --url https://github.com/developer-guy/desired-state-repository \
+    --branch master \
+    --interval 30s
+```
+
+To create `Kustomization`:
+
+```shell
+flux create kustomization alpine \
+    --source alpine \
+    --path "./" \
+    --prune true \
+    --validation client \
+    --interval 5m
+```
+
+### Install falco-event-listener
+
+As the name suggests, this program will listen to an event in the form of `CloudEvents.` This CloudEvents forwarded from Falcosidekick, a simple daemon for enhancing available outputs for Falco. After the successful receipt of the event, `falco-event-listener` will update the YAML definition to scale its replicas to zero based on the pieces of information given in the event.
+
+In most basic form, the architecture of the demo is:
+
+Falco w/HTTP --> Falcosidekick w/CloudEvent --> falco-event-listener w/HTTP --> GitHub
+
+To learn more about `CloudEvents`, see [cloudevents.io](https://cloudevents.io).
+
+First, let us clone the falco-event-listener repository.
```

Comment:
```suggestion
First, let us clone the `falco-event-listener` repository.
```

Reply:
thank you so much for fixing the issues, it should be fixed now 🙏